src/youtube/js/youtube.js

```diff
@@ -366,9 +366,13 @@ require("../../../html5-common/js/utils/constants.js");
         youtubeID = '';
       }
       player = null;
-      youtubePlayer.destroy();
-      youtubePlayer = null;
+      if (youtubePlayer) {
+        youtubePlayer.destroy();
+        youtubePlayer = null;
+      }
       playerReady = false;
+      clearInterval(timeUpdateInterval);
```

Comment:
No, it doesn't.  I just verified (pause youtube video and switch to another youtube video).  In general – clearing time update that was previously cleared is harmless